docs/guide/breakpoint-preview.md

```diff
@@ -84,100 +84,95 @@ Each of the individual breakpoint properties in the `screen` object take a `labe
 
 ---
 
-### Limitations and Differences
+## Limitations and Differences
 While the ApostropheCMS breakpoint preview effectively converts many media queries to container queries, there are some key limitations and differences to be aware of. These limitations stem from fundamental differences between **media queries** (which target the viewport) and **container queries** (which target individual containers).
 
 #### 1. **Unsupported Parameters**
-Certain media query conditions are not supported or do not translate well to container queries:
+Certain media query conditions do not translate well to container queries:
 
 - **Orientation queries**:
-  - Parameters like `orientation: landscape` or `orientation: portrait` do not apply to containers, as containers lack inherent orientation.
-  - Example of unsupported query:
+  - Parameters like `orientation: landscape` or `orientation: portrait` don't have equivalent behavior in container queries, as containers don't have the same orientation concept as viewports.
+  - Example of query with limited support:
```

Comment:
Limited how?

Reply:
Yeah, reading with fresh eyes, that wording is a little off